Resources/Prototypes/Radium/radio_channels.yml

```
@@ -0,0 +1,7 @@
+- type: radioChannel
+  id: Vox
+  name: chat-radio-vox
+  keycode: 'v' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)